src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/PreparedStatementTest.java

```diff
@@ -170,6 +170,8 @@ public void testStatementPooling() throws SQLException {
             }
         }
 
+        Utils.skipTestForSQLServer2008((SQLServerConnection) DriverManager.getConnection(connectionString));
```

Comment:
Hello @TobiasSQL, just want to confirm with you. If we change throw to RAISERROR , the test should still be valid, right ?